tools/markdown-checker/lib/analyzer/detectHeaders.js

```diff
@@ -1,10 +1,10 @@
-const traverseChildren = require("./../transformer/traverser").traverseChildren;
+const { traverseChildren } = require('./../transformer/traverser');
 /**
  * @param {Object} tableRow
- * @returns {Object} - collected header template 
+ * @returns {Object} - collected header template
  */
 module.exports = function detectHeaders(tableRow) {
-  let cellSize = tableRow.align.length;
-  let [headerNode] =  tableRow.children;
-  //create header template here
+  const cellSize = tableRow.align.length;
```

Comment:
`detectHeaders` fonksiyonu birden çok **row** için yeniden çalışıyor. `return` de vermedim henüz. `debugging` yaparken manipüle edebiliyorum bu değişkenleri ben.  Şimdilik `let` kalsın burası, işim bitince değiştireceğim ben.

Reply:
geri aldım.